test/with-typescript/index.ts

```diff
@@ -1065,11 +1070,20 @@ const cookieHandlerInput: CookieHandlerInput = (original: CookieHandlerInterface
     };
 };
 
+const dateProviderImplementation: DateProviderInterface = {
+    getClientClockSkewInMillis: () => 0,
+    setClientClockSkewInMillis: () => {},
+    now: () => Date.now(),
+};
+
+const dateProviderInput: DateProviderInput = () => dateProviderImplementation;
+
 SuperTokens.init({
     appInfo,
     recipeList,
```

Comment:
add a test where we init web-js with custom datepicker and then check that that is being used.

Reply:
done